src/base/memory_monitor_sr.cpp

```diff
@@ -588,6 +599,15 @@ namespace cubperf
   {
     add_stat (src, - (size));
     add_stat (dest, size);
+
+    /* If move is occurred from other modules to MMON_COMMON,
+     * it means that memory is returned to common structures for keeping.
+     * ex) pool, free list, reserved, etc..
+     * In this case, transaction's memory usage will be subtracted. */
+    if (dest == MMON_COMMON)
```

Comment:
The `if (src == MMON_COMMON)` condition will be addressed in following commit. 
it is discussed offline